loafer/runners.py

```diff
@@ -43,7 +43,7 @@ def prepare_stop(self, *args):
             self.loop.stop()
 
     def _cancel_all_tasks(self):
-        to_cancel = asyncio.Task.all_tasks(self.loop)
+        to_cancel = asyncio.all_tasks(self.loop)
```

Comment:
```The asyncio.Task.current_task() and asyncio.Task.all_tasks() have been removed. They were deprecated since Python 3.7 and you can use asyncio.current_task() and asyncio.all_tasks() instead. (Contributed by Rémi Lapeyre in bpo-40967)
```
https://docs.python.org/3/whatsnew/3.9.html#removed